aspnetcore/fundamentals/host/web-host.md

```diff
@@ -51,6 +45,8 @@ public class Program
 }
 ```
 
+The code that calls `CreateDefaultBuilder` is in a method named `CreateWebHostBuilder`, which separates it from the code in `Main` that calls `Run` on the builder object. This separation is required if you use [Entity Framework Core tools](/ef/core/miscellaneous/cli/). The tools expect to find a `CreateWebHostBuilder` method that they can call at design time to configure the host without running the app. An alternative is to implement `IDesignTimeDbContextFactory`. For more information, see [Design-time DbContext Creation](/ef/core/miscellaneous/cli/dbcontext-creation).
```

Comment:
@divega Please review this paragraph -- This is related to the EF Core doc issue  https://github.com/aspnet/EntityFramework.Docs/issues/1465